packages/playwright-ct-core/types/component.d.ts

```diff
@@ -46,7 +46,11 @@ declare global {
     playwrightMount(component: Component, rootElement: Element, hooksConfig?: any): Promise<void>;
     playwrightUnmount(rootElement: Element): Promise<void>;
     playwrightUpdate(rootElement: Element, component: Component): Promise<void>;
-    __pw_hooks_before_mount?: (<HooksConfig>(params: { hooksConfig?: HooksConfig; } & any) => Promise<any>)[];
-    __pw_hooks_after_mount?: (<HooksConfig>(params: { hooksConfig?: HooksConfig; } & any) => Promise<void>)[];
+    __pw_hooks_before_mount?: (<HooksConfig extends JsonObject = JsonObject>(
+      params: { hooksConfig?: HooksConfig; [key: string]: any }
```

Comment:
You are using Record<string, any> in other places.

Reply:
I don't know if this can be done in another way? This is invalid syntax:

```ts 
__pw_hooks_before_mount?: (<HooksConfig>(params: { hooksConfig?: HooksConfig; Record<string, any> }) => Promise<any>)[];
```